DolbyIO/Source/Public/DolbyIOFunctions.h

```diff
@@ -380,4 +487,20 @@ class UDolbyIOBlueprintFunctionLibrary : public UBlueprintFunctionLibrary
 	static void SetLogSettings(const UObject* WorldContextObject, EDolbyIOLogLevel SdkLogLevel = EDolbyIOLogLevel::Info,
 	                           EDolbyIOLogLevel MediaLogLevel = EDolbyIOLogLevel::Off,
 	                           const FString& LogDirectory = "");
+
+	/** Sets preferred audio input device.
+	 *
+	 * @param NativeId - The string representing native id of wanted device .
```

Comment:
Unnecessary space before the period and I would just say `The native ID of the device.` again. id sounds like we're being philisophical and wanted sounds like the device did something wrong ;) Same below.